packages/slate-editor/src/extensions/button-block/ButtonBlockNode.ts

```diff
@@ -0,0 +1,39 @@
+import type { ElementNode } from '@prezly/slate-types';
+import { isElementNode } from '@prezly/slate-types';
+import type { Node } from 'slate';
+
+type Uuid = string;
+
+export interface ButtonBlockNode extends ElementNode {
+    type: typeof ButtonBlockNode.Type;
+    uuid: Uuid;
+    href: string;
+    layout: ButtonBlockNode.ButtonLayout;
+    variant: ButtonBlockNode.ButtonVariant;
```

Comment:
Followup on the previous comment:

```suggestion
    layout: `${ButtonBlockNode.Layout}`;
    variant: `${ButtonBlockNode.Variant}`;
```